components/desktop/gnome2/gnome-games/Makefile

```diff
@@ -29,7 +30,8 @@ include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+# Need automake 1.11
```

Comment:
Just to make clear that you do not need to open a new PR: modify the branch and push -f to overwrite the content, the PR will be updated accordingly.